resources/env/delivery/tomcat-config/crafter/social/management/extension/server-config.properties

```diff
@@ -1 +1 @@
-crafter.profile.rest.client.url.base=http://${tomcat.host}:${tomcat.http.port}/crafter-profile
+crafter.profile.rest.client.url.base=http://localhost:${tomcat.http.port}/crafter-profile
```

Comment:
So the location of profile can't come from env like other URLs?

Reply:
Latest commit fixes that, now all the URLs can be specified as env variables.